src/tests/ftest/process_core_files.py

```diff
@@ -260,7 +262,7 @@ def install_debuginfo_packages(self):
             cmds.append(
                 ["sudo", "dnf", "debuginfo-install", "-y"] + dnf_args
                 + ["daos-client-" + rpm_version, "daos-server-" + rpm_version,
-                   "daos-tests-" + rpm_version])
+                   "daos-client-tests-" + rpm_version, "daos-server-tests-" + rpm_version])
```

Comment:
Should we be more inclusive here and install all `daos-*-debuginfo` files?  In this PR for example I see:

daos-admin-debuginfo-2.3.103-1.9095.g3cbc7ce0.el8.x86_64.rpm
daos-client-debuginfo-2.3.103-1.9095.g3cbc7ce0.el8.x86_64.rpm
daos-client-tests-debuginfo-2.3.103-1.9095.g3cbc7ce0.el8.x86_64.rpm
daos-client-tests-openmpi-debuginfo-2.3.103-1.9095.g3cbc7ce0.el8.x86_64.rpm
daos-debuginfo-2.3.103-1.9095.g3cbc7ce0.el8.x86_64.rpm
daos-firmware-debuginfo-2.3.103-1.9095.g3cbc7ce0.el8.x86_64.rpm
daos-serialize-debuginfo-2.3.103-1.9095.g3cbc7ce0.el8.x86_64.rpm
daos-server-debuginfo-2.3.103-1.9095.g3cbc7ce0.el8.x86_64.rpm
daos-server-tests-debuginfo-2.3.103-1.9095.g3cbc7ce0.el8.x86_64.rpm

It's probably only a matter of time when something from one of the ones not being installed segfaults and somebody wants a stacktrace from it.

Reply:
Thanks, added.